### ISSUE TITLE
[FEATURE] : Changing from lucide-react to other alternative icon library 

### ISSUE THREAD

**Issue by TenzDelek** [asyncapi/conference-website#471]
### Why do we need this improvement?

currently we are having lucide react as our icon library. we are only implementing at only one place 
<img width="230" alt="image" src="https://github.com/user-attachments/assets/42d69e28-9911-44d6-bc15-0acb1d7cc5e6">

currently I am implementing this #469 and was trying to use the lucide react icon for socials like github and all but it turns out that they have deprecated those
<img width="518" alt="image" src="https://github.com/user-attachments/assets/032b0e96-df0c-4f38-9353-fcea482761a1">

in light of all this limitation, I am prosposing to replace lucide and use other alternative like [react icon](https://react-icons.github.io/react-icons/) which have a diverse set of icon compare to lucide and heroicon.

would love to hear from @AceTheCreator , @ashmit-coder about this changes.

### How will this change help?

more set of diverse icon 

### How could it be implemented/designed?

npm

### 🚧 Breaking changes

Yes

### 👀 Have you checked for similar open issues?

- [X] I checked and didn't find a similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md)

### Are you willing to work on this issue?

Yes I am willing to submit a PR!

**Comment by TenzDelek** [comment]
cc @AceTheCreator , @ashmit-coder

**Comment by shivasankaran18** [comment]
can I work on this issue sir..?

**Comment by TenzDelek** [comment]
no need to raise seperate pr for this. will be implemented in #469 if approved

**Comment by TenzDelek** [comment]
> @TenzDelek i think using an icon library is a bit redundant for us, as we are not using a lot of them. I think using the SVG of this icons in the /illustration will be ideal, don't you think? 

Agree with you, will raise a pr soon. Should we consider removing the icon library(as it will be un used) or keep it for the future sake.

**Comment by AceTheCreator** [comment]
I'm closing this issue. An ideal issue for this will be removing the package

**Comment by TenzDelek** [comment]
> I'm closing this issue. An ideal issue for this will be removing the package

true, but currently, there are some component (ticketcard,pastedition) that uses lucide react. so removing the package and adding manual svg would be a bad choice. so thats why i closed that pr.
